ProjectHive/ProjectHive.Services.ProjectsAPI/Business/Services/ProjectService.cs

```diff
@@ -1,6 +1,64 @@
-﻿namespace ProjectHive.Services.ProjectsAPI.Business.Services
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using ProjectHive.Services.ProjectsAPI.Data.Entities;
+using ProjectHive.Services.ProjectsAPI.Data.Repository.Interfase;
+using ProjectHive.Services.ProjectsAPI.Dto;
+
+namespace ProjectHive.Services.ProjectsAPI.Business.Services
 {
     public class ProjectService : IProjectService
     {
+        private readonly IUnitOfWork _unitOfWork;
+        private readonly IMapper _mapper;
+        public ProjectService(IUnitOfWork unitOfWork, IMapper mapper)
+        {
+            _unitOfWork = unitOfWork;
+            _mapper = mapper;
+        }
+        public async Task<ProjectDto?> GetProjectById(Guid Id)
+        {
+            var project= _mapper.Map<ProjectDto>(await _unitOfWork.ProjectRepository.GetById(Id));
+            return project;
+        }
+        public async Task DeleteProjectById(Guid Id)
+        {
+            await _unitOfWork.ProjectRepository.DeleteById(Id);
+            await _unitOfWork.Commit();
+        }
+
+        public async Task CreateProject(ProjectDto dto)
+        {
+            var project = _mapper.Map<Project>(dto);
+            await _unitOfWork.ProjectRepository.CreateOne(project);
+            await _unitOfWork.Commit();
+        }
+
```

Comment:
There and below better to return created or updated project or projects
`public async Task<ProjectDto > CreateProject(ProjectDto dto)` or something like that